__tests__/Auth0Client.test.ts

```diff
@@ -1140,15 +1154,208 @@ describe('Auth0Client', () => {
         }
       });
 
+      jest.spyOn(<any>utils, 'runIframe');
+
+      mockFetch.mockReset();
+
+      const token = await auth0.getTokenSilently();
+
+      expect(token).toBe(TEST_ACCESS_TOKEN);
+      expect(utils.runIframe).not.toHaveBeenCalled();
+      expect(mockFetch).not.toHaveBeenCalled();
+    });
+
+    it('calls the authorize endpoint using the correct params', async () => {
+      const auth0 = setup();
+
+      jest.spyOn(<any>utils, 'runIframe').mockResolvedValue({
+        access_token: TEST_ACCESS_TOKEN,
+        state: TEST_STATE
+      });
```

Comment:
In the other tests for `getTokenWithPopup`, I've been able to use `runIframe` without mocking and instead make use of a new function that abstracts the message handler, do you think we could do the same thing here?

https://github.com/auth0/auth0-spa-js/blob/fd76b2cdc3333b4413a21fc18edd98662a5a0908/__tests__/Auth0Client.helpers.ts#L185-L206

Reply:
Good call, I will give it a try.

Reply:
Hmmm, I can not seem to get this working. The iframe here is different as to other places. This test wants to verify that the currect /authorizeUrl is used. However, we are not redirecting the application but using an iframe that only exists for a short duration.

We could start mocking the iframe element (window.document.createElement etc), but I feel like this might be a bit to complex. What do you think?

Reply:
That's fine, let's leave it for now if it's different.